packages/backend/src/api/resources/resourceModel.ts

```diff
@@ -4,13 +4,20 @@ import { z } from 'zod';
 extendZodWithOpenApi(z);
 
 export type Resource = z.infer<typeof ResourceSchema>;
- export const ResourceSchema = z.object({
-   id: z.string().uuid(),
-   title: z.string().min(2, 'Title cannot be empty').max(100, 'Title is too long'),
-   description: z.string().min(10, 'Description is too short').max(5000, 'Description is too long'),
-   link: z.string().url('Link must be a valid URL'),
-   publish: z.boolean().default(false),
-   tags: z.string(),
-   type: z.string(),
- });
+export const ResourceSchema = z.object({
+  id: z.string().uuid(),
+  title: z.string().min(2, 'Title cannot be empty').max(100, 'Title is too long'),
+  description: z.string().min(10, 'Description is too short').max(5000, 'Description is too long'),
+  link: z.string().url('Link must be a valid URL'),
+  publish: z.boolean().default(false),
+  tags: z.string(),
+  type: z.string(),
+});
 
+export type SavedResource = z.infer<typeof SavedResourceSchema>;
+export const SavedResourceSchema = z.object({
+  id: z.number(),
+  resourceId: z.string().uuid(),
+  userId: z.string().uuid(),
```

Comment:
shouldnt we use stringId in common validations?

Reply:
actually I didn't understand the `stringId` it said that Id must be numeric value and num >0, and a uuid isnt a numeric value

Reply:
ohh you're right i mixed up, keep it like this 👍